html/themes/custom/common_design_subtheme/components/rw-river-letter-navigation/rw-river-letter-navigation.js

```diff
@@ -0,0 +1,27 @@
+(function () {
```

Comment:
Note: I'm not adding `Drupal` because technically this scripts doesn't require it.